hallux/backends/dummy_backend.py

```diff
@@ -51,7 +51,6 @@ def __del__(self):
 
     def report_succesfull_fix(self, issue: IssueDescriptor, proposal: DiffProposal) -> None:
         hash = self.issue_hash(issue.description, proposal.issue_lines)
-
         if hash not in self.json:
             self.json[hash] = "\n".join(proposal.proposed_lines)
```

Comment:
Rename this variable; it shadows a builtin.
```suggestion
        issue_hash = self.issue_hash(issue.description, proposal.issue_lines)
        if issue_hash not in self.json:
            self.json[issue_hash] = "\n".join(proposal.proposed_lines)
```